changelog/7790.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed incorrect URI handling in `~sunpy.map.Map` and `~sunpy.timeseries.TimeSeries`.
```

Comment:
I feel like this is a new feature, instead of a bugfix.

Reply:
renamed file to `changelog/7790.feature.rst`